#!/usr/bin/env python
PACKAGE = "differential_drive"
import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("number_of_readings_to_average", int_t, 0x01, "Number of velocity readings to average", 8, 1, 64)
gen.add("percent_velocity_mismatch_allowed", double_t, 0x02, "Percent that the velocity reading have to differ by to be flagged", 10.0, 0.0, 100.0)
gen.add("stasis_lower_velocity_limit", double_t, 0x04, "Velocity under which mismatches are ignored", 0.001, 0.0, 0.5)

exit(gen.generate(PACKAGE, "differential_drive", "MovementStatusParameters"))
